scripts/acbuild-rkt-builder.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -ex
+
+MODIFY=${MODIFY:-""}
+FLAGS=${FLAGS:-""}
+IMG=${IMG:-"debian"}
+IMGVERSION=${VERSION:-"sid"}
+DOCKERIMG="$IMG:$IMGVERSION"
+ACIFILE=${ACIFILE:-"$IMG-$IMGVERSION.aci"}
+OUTACI="rkt-builder.aci"
+BUILDDIR=/opt/build-rkt
+SRC_DIR=/opt/rkt
+ACI_GOPATH=/go
+VERSION=${VERSION:-"v0.10.0"}
+# if [ `expr match $VERSION '.*git$'` -gt 0 ]; then
+#     VERSION="$VERSION$(git rev-parse --short HEAD 2>/dev/null)$(git diff-index --quiet HEAD 2>/dev/null || echo -dirty)"
+# fi
+echo "Version: $VERSION"
+
+echo "Building $ACIFILE"
+
+which docker2aci || go get github.com/appc/docker2aci
+if ! [ -e "$ACIFILE" ]; then
+    docker2aci "docker://$DOCKERIMG"
+    # These base images dont always come with valid values
+    actool patch-manifest -user 0 -group 0 --name $IMG-$IMGVERSION --exec /bin/bash --replace $ACIFILE
+fi
+
+# If modify is specified, pass the modify flag to each command and don't use
+# acbuild begin, write or end otherwise build with a context, and setup a trap
+# to handle failures,
+if ! [ -z "$MODIFY" ]; then
+    FLAGS="--modify $MODIFY $FLAGS"
+    OUTACI=$MODIFY
+else
+    trap "{ export EXT=$?; acbuild end && exit $EXT; }" EXIT
+    acbuild $FLAGS begin "$ACIFILE"
+fi
+
+acbuild $FLAGS set-name coreos.com/rkt/builder
+acbuild $FLAGS label add version $VERSION
+acbuild $FLAGS set-user 0
+acbuild $FLAGS set-group 0
+acbuild $FLAGS environment add OS_VERSION $IMGVERSION
+acbuild $FLAGS environment add GOPATH $ACI_GOPATH
+acbuild $FLAGS environment add BUILDDIR $BUILDDIR
+acbuild $FLAGS environment add SRC_DIR $SRC_DIR
+acbuild $FLAGS mount add build-dir $BUILDDIR
+acbuild $FLAGS mount add src-dir $SRC_DIR
+acbuild $FLAGS copy $PWD/scripts /scripts
+acbuild $FLAGS run /bin/bash /scripts/install-deps-$IMG-$IMGVERSION.sh
+acbuild $FLAGS run /bin/bash /scripts/install-appc-spec.sh
+acbuild $FLAGS set-exec "/bin/bash -c 'cd /opt/rkt && ./autogen.sh && ./configure && make'"
+if [ -z "$MODIFY" ]; then
+    acbuild $FLAGS write --overwrite $OUTACI
+fi
+
+echo "Usage: rkt run --insecure-skip-verify --interactive --volume src-dir,kind=host,source=/path/to/rkt --volume build-dir,kind=host,source=/path/to/rkt-build $OUTACI --exec /bin/bash -- -c 'cd $SRC_DIR && ./autogen.sh && ./configure && make'"
```

Comment:
No, I put it to this just so people have an idea of what to do if they want to run the build manually or tweak it.